package.json

```diff
@@ -85,6 +85,11 @@
         "title": "Show in Memory Inspector",
         "category": "Memory"
       },
+      {
+        "command": "memory-inspector.go-to-value",
+        "title": "Go to value in Memory Inspector",
```

Comment:
```suggestion
        "title": "Go to Value in Memory Inspector",
```

Reply:
In fact I am wondering if `Show Value in Memory Inspector` would be more consistent with `Show in Memory Inspector`.

Reply:
I was thinking "Go to value" would be clearer than "Show value", since the latter might be interpreted as toggling the data column. For the moment I'll just update the capitalization.

Reply:
OK, understood. I am good with that.